detox/e2e/test/autocomplete/edit_post.e2e.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// *******************************************************************
+// - [#] indicates a test step (e.g. # Go to a screen)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element testID when selecting an element. Create one if none.
+// *******************************************************************
+
+import {
+    Post,
+    Setup,
+} from '@support/server_api';
+import {
+    serverOneUrl,
+    siteOneUrl,
+} from '@support/test_config';
+import {Autocomplete} from '@support/ui/component';
+import {
+    ChannelListScreen,
+    ChannelScreen,
+    EditPostScreen,
+    HomeScreen,
+    LoginScreen,
+    ServerScreen,
+} from '@support/ui/screen';
+import {getRandomId} from '@support/utils';
+import {expect} from 'detox';
+
+describe('Autocomplete - Edit Post', () => {
+    const serverOneDisplayName = 'Server 1';
+    const channelsCategory = 'channels';
+
+    beforeAll(async () => {
+        const {channel, user} = await Setup.apiInit(siteOneUrl);
+
+        // # Log in to server
+        await ServerScreen.connectToServer(serverOneUrl, serverOneDisplayName);
+        await LoginScreen.login(user);
+
+        // * Verify on channel list screen
+        await ChannelListScreen.toBeVisible();
+
+        // # Open a channel screen, post a message, open post options for message, and open edit post screen
+        const message = `Messsage ${getRandomId()}`;
+        await ChannelScreen.open(channelsCategory, channel.name);
+        await ChannelScreen.postMessage(message);
+        const {post} = await Post.apiGetLastPostInChannel(siteOneUrl, channel.id);
+        await ChannelScreen.openPostOptionsFor(post.id, message);
+        await EditPostScreen.open();
+    });
+
+    beforeEach(async () => {
+        // # Clear message input
+        await EditPostScreen.messageInput.clearText();
+    });
+
+    afterAll(async () => {
+        // # Log out
+        await EditPostScreen.close();
+        await ChannelScreen.back();
+        await HomeScreen.logout();
+    });
+
```

Comment:
For this and others... should we make sure the autocomplete component is completely inside the device screen? (Not sure if supported by this)